modules/Miss.js

```diff
@@ -40,12 +40,18 @@ class Miss extends React.Component {
     return (
       <LocationSubscriber>
         {(location) => {
-          const { render, component:Component } = this.props
+          const { children, render, component:Component } = this.props
           const { noMatchesInContext } = this.state
           const { serverRouter, match } = this.context
           const noMatchesOnServerContext = serverRouter &&
             serverRouter.missedAtIndex(match.serverRouterIndex)
-          if (noMatchesInContext || noMatchesOnServerContext) {
+          const matched = !(noMatchesInContext || noMatchesOnServerContext)
```

Comment:
The idea is that `children` should let you accomplish the same thing as `render`, but allowing the user to control what happens when it should or shouldn't render.  The example in the docs is for animation, where a wrapper needs to always be rendered but it only has children if it's a match.